go.mod

```diff
@@ -2,4 +2,10 @@ module github.com/beyondstorage/go-service-ipfs
 
 go 1.15
 
-require github.com/beyondstorage/go-storage/v4 v4.2.0
+require (
+	github.com/beyondstorage/go-endpoint v1.0.1
+	github.com/beyondstorage/go-integration-test/v4 v4.1.2-0.20210701072246-3339bf2736ac
```

Comment:
Ping @JinnyYi to release a new version of `go-integration-test` (maybe after https://github.com/beyondstorage/specs/pull/134 get merged?)

Reply:
Got it. I'll update `go-integration-test` if needed and release a new version after https://github.com/beyondstorage/specs/pull/134 get merged.